src/components/dialogs/parameters/widget/ParameterLine.tsx

```diff
@@ -0,0 +1,170 @@
+                        <FormattedMessage id={props.labelTitle} />
+                        {/*</Typography>*/}
+                    </Grid>
+                    <Grid item container xs={4} className={classes.controlItem}>
+                        <Select
+                            labelId={props.labelValue}
+                            value={
+                                props.defaultValueIfNull && !parameterValue
+                                    ? Object.entries<string>(props.values)[0]
+                                    : parameterValue
+                            }
+                            onChange={(event) => {
+                                handleChangeParameterValue(event.target.value);
+                            }}
+                            size="small"
+                            disabled={props.disabled ?? false}
+                        >
+                            {Object.entries<string>(props.values).map(
+                                ([key, value]) => (
+                                    <MenuItem key={key} value={key}>
+                                        <FormattedMessage id={value} />
+                                    </MenuItem>
+                                )
+                            )}
+                        </Select>
+                    </Grid>
+                </>
+            );
+
+        case ParameterType.Slider:
+            // eslint-disable-next-line react-hooks/rules-of-hooks
+            const [sliderValue, setSliderValue] = useState(
+                Number(parameterValue)
+            );
+            return (
+                <>
+                    <Grid item xs={8} className={classes.parameterName}>
+                        {/*<Typography component="span" variant="body1" fontWeight="fontWeightBold" margin={1}>*/}
+                        <FormattedMessage id={props.label} />
+                        {/*</Typography>*/}
+                    </Grid>
+                    <Grid item container xs={4} className={classes.controlItem}>
+                        <Slider
+                            min={props.minValue ?? 0}
+                            max={props.maxValue ?? 100}
+                            valueLabelDisplay="auto"
+                            onChange={(event, newValue) => {
+                                // @ts-ignore
+                                setSliderValue(newValue);
+                            }}
+                            onChangeCommitted={(event, value) => {
+                                handleChangeParameterValue(value);
+                            }}
+                            value={sliderValue}
+                            disabled={props.disabled ?? false}
+                            marks={props.marks}
+                        />
+                    </Grid>
+                </>
+            );
+
+        default:
+            //throw new Error(`Not supported parameter type ${type}`);
```

Comment:
it's more because I haven't decide for now how to support this case: do I insert a simple fallback text or do an error? If the second, as a throw or just as a log without blocking ?

It's more for when used from JS files as there don't seem to be type-checking for js.